components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss

```diff
@@ -25,13 +25,17 @@ chef-icon.SKIPPED {
   color: $chef-dark-grey;
 }
 
+.status-icon {
+
+  &--pointer {
+    cursor: pointer;
+  }
+}
+
 .run-info-tooltip {
   border-left: $spacing-03 solid;
   border-right: $spacing-03 solid transparent;
   padding-right: $spacing-03;
-  bottom: 100%;
-  left: 50%;
-  margin-left: -60px; 
```

Comment:
tooltip placement is generated by the `chef-tooltip` component, so adding our own placement on top of that will throw it off.

Reply:
that makes sense!